code/src/index.css

```diff
@@ -1,13 +1,108 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: 'Trispace', sans-serif;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.thought-card, .thoughts-form {
+  margin: 20px auto;
+  border: 2px solid black;
+  width: 500px;
+  box-shadow: 5px 5px;
+  padding: 20px;
 }
+
+.thought-card {
+  word-wrap: break-word;
+}
+
+.thoughts-form {
+  background-color: #f2f0f0;
+  display: flex;
+  flex-direction: column;
+}
+
+.form-title {
+  margin-top: 0;
+  font-weight: bold;
+}
+
+.form-button {
+  margin-top: 10px;
+  width: 250px;
+  background-color: #ffadad;
+  border: none;
+  padding: 10px;
+  border-radius: 30px;
+  font-family: 'Trispace', sans-serif;
+  font-weight: bold;
+  cursor: pointer;
+}
+
+textarea {
```

Comment:
Good job with styling of `<textarea />`, I tried to break your styles by resizing it, but you styled it very well ;)